Source/WTF/wtf/posix/FileSystemPOSIX.cpp

```diff
@@ -270,15 +271,17 @@ std::pair<String, PlatformFileHandle> openTemporaryFile(StringView prefix, Strin
     // This is OK for now since the code using it is built on macOS only.
     ASSERT_UNUSED(suffix, suffix.isEmpty());
 
-    char buffer[PATH_MAX];
-    if (snprintf(buffer, PATH_MAX, "%s/%sXXXXXX", temporaryFileDirectory(), prefix.utf8().data()) >= PATH_MAX)
-        goto end;
+    const char* tmp = temporaryFileDirectory();
```

Comment:
In WebKit coding style we’d typically name this something more like `directory`.